api/v1/cosmosfullnode_types.go

```diff
@@ -361,6 +377,16 @@ type ChainConfig struct {
 	// $DATA_DIR: The directory for the database files.
 	// +optional
 	SnapshotScript *string `json:"snapshotScript"`
+
+	// If configured as a Sentry, invokes sleep command with this value before running chain start command.
+	// Currently, tendermint requires the privval laddr to be available immediately without any retry.
+	// This workaround gives time for the connection to be made to a remote signer.
+	// If a Sentry and not set, defaults to 10.
+	// If set to 0, omits injecting sleep command.
+	// Assumes chain image has `sleep` in $PATH.
+	// +kubebuilder:validation:Minimum:=0
+	// +optional
+	PrivvalSleepSeconds *int32 `json:"privvalSleepSeconds"`
```

Comment:
Is there any reason we would ever want it to be something other than 10 for a sentry?

Reply:
This is future proofing for other k8s providers, like AWS and Azure. 10 seconds works for us on GKE with our configuration, but it may not work for other cloud providers. This gives the user the ability to tweak if needed.